src/styles/Blocks/Header.scss

```diff
@@ -0,0 +1,66 @@
+header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
```

Comment:
- Create a separate file for each BEM block
- Use `scss` nesting, fix in all places